### DISCUSSION TITLE
Missing transactions in finances

### DISCUSSION THREAD

**Discussion by masterevgeni** [amzn/selling-partner-api-models#3356] [General]
Hello dear Amazon support,
I am a developer in Celor company.
We are trying to get financial data by using sp-api for api queries.
While querying /finances/v0/financial for events between date X and date Y, we receive a response with missing transactions.
For example we have 5 transactions, the API returns transactions 1,2,3,5 without transaction #4.
It does not reproduce constantly, but daily.
What we can do?
Thanks,
Evgani, Celor team (edited)

**Comment by tspicer** [comment]
@masterevgeni did you ever get a resolution to this issue? I see this occurring as well, and it is difficult to troubleshoot
